backend/prompt_studio/prompt_studio_core/prompt_studio_helper.py

```diff
@@ -654,18 +677,20 @@ def dynamic_indexer(
             profile_manager.chunk_size = 0
 
         try:
+            usage_kwargs = {"run_id": run_id}
             util = PromptIdeBaseTool(log_level=LogLevel.INFO, org_id=org_id)
             tool_index = ToolIndex(tool=util)
```

Comment:
@gaya3-zipstack Shouldn't we change this `ToolIndex` also to `Index`?

We could do a quick grep for similar constructs if that helps. This could ensure all locations are updated to new SDK interface.

Reply:
Thanks a lot @hari-kuriakose  Done. Did a grep across cloud, prompt_studio and backend . Looks clean.